include/igl/read_triangle_mesh.cpp

```diff
@@ -43,7 +43,7 @@ IGL_INLINE bool igl::read_triangle_mesh(
     bool success = readOBJ(str,V,TC,N,F,FTC,FN);
     for(auto & v : V)
     {
-      v.resize(std::min(v.size(),(size_t)3));
+      v.resize((std::min)(v.size(),(size_t)3));
```

Comment:
I guess so, that's the safest way crossplatform-wise. Or you ask all the windows people to define NOMINMAX before including there windows.h in their projects.